metrics/report/README.md

```diff
@@ -0,0 +1,61 @@
+# Kata Containers metrics report generator
+
+The files within this directory can be used to generate a 'metrics report'
+for Kata Containers.
+
+The primary workflow consists of two stages:
+
+1) Run the provided report metrics data gathering scripts on the system(s) you wish
+to analyse
+2) Run the provided report generation script to analyse the data and generate a
+report file.
+
+## Data gathering
+
+Data gathering is provided by the `grabdata.sh` script. When run, this script will
+execute a set of tests from the `tests/metrics` directory. The resulting JSON results files
+will be placed into the `tests/metrics/results` directory.
+
+Once the results have been generated, create a suitably named subdirectory of
+`tests/metrics/results`, and move the JSON files into it.
+
+Repeat this process if you wish to compare multiple sets of results. Note, the
+report generation scripts will process all subfolders of `tests/metrics/results' when
```

Comment:
I think you need to replace the apostrophe before "when" with a backtick to fix the formatting?